### PR TITLE
Add interactive flag

### PR DESCRIPTION
**Pull request by erik**
```
$ ./bin/sugar ssh 'Foo' -n                                                           
foo matches multiple different instances.
0: i-aaaaa   foo 2       ...us-west-2.compute.amazonaws.com
1: i-bbbbb   foo 3       ...us-west-2.compute.amazonaws.com
2: i-ccccc   foo 1       ...us-west-2.compute.amazonaws.com
Select an instance: -1
Select an instance: 2
Connecting to i-ccccc (one of 3 instances matching "foo")
```

### CODE REVIEW

File: src/sugar.js [hunk 1]
```diff
@@ -178,16 +184,16 @@ function getInstances(ec2) {
 }
 
 function listInstances(instances) {
-  instances.forEach(function (instance) {
+  instances.forEach(function (instance, index) {
     console.log([
-      instance.InstanceId,
+      `${index}: ${instance.InstanceId}`,
```
Comment (avram) [inline src/sugar.js]: If we're not in interactive mode, why would we want to print an index?

File: src/sugar.js [hunk 2]
```diff
@@ -81,6 +82,11 @@ addCommonSSHOpts(
 
 addCommonSSHOpts(
   opts.command('ssh')
+    .option('interactive', {
+      abbr: 'n',
+      flag: true,
+      help: 'Prompt the user to select a specific instance'
```
Comment (avram) [inline src/sugar.js]: Might this mention that it is interactive solely if there are multiple matches?

File: src/sugar.js (hunk 1 above)
Comment (erik) [inline src/sugar.js]: Mainly my laziness :smile:

**Comment by avram** [comment]
:+1:  here. @danopia might want to chime in as the most sugary person here, but looks useful + correct!

**Pull request merged**